ports/opencascade/portfile.cmake

```
@@ -45,6 +46,30 @@ vcpkg_install_cmake()
 
 vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/opencascade TARGET_PATH share/opencascade)
 
+
```

```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/opencascade)
```
`TARGET_PATH ` will specify to `share/${PORT}` by default. So there is no need to add this argument.